src/test/java/io/airlift/airline/ArgConv.java

```diff
@@ -0,0 +1,115 @@
+package io.airlift.airline;
+
+public class ArgConv {
+
+    public static void main(String... args) {
+        Cli.CliBuilder<Runnable> builder = Cli.<Runnable>builder("argtest")
+                .withDescription("test the argument converter")
+                .withDefaultCommand(Help.class)
+                .withCommand(Const.class)
+                .withCommand(ValueOf.class)
```

Comment:
Do we need subcommands?
I think we could have just a bag of (optional) parameters and let a test method just set one argument.
like `"--value-of", "12345"`

This would reduce number of classes needed.